apps/web-giddh/src/app/models/api-models/Subscriptions.ts

```diff
@@ -46,7 +47,7 @@ export class PlanDetails {
     createdAt: string;
     ratePerExtraTransaction: number;
     isCommonPlan: boolean;
-    duration: number;
+    duration: any;
```

Comment:
In line number 35 you took duration as string but here you take any, please check